composer.json

```diff
@@ -67,7 +71,8 @@
 
 		"symfony/browser-kit": "~2.7",
 		"silex/web-profiler": "~2.0",
-		"sorien/silex-dbal-profiler": "~2.0"
+		"sorien/silex-dbal-profiler": "~2.0",
+		"wmde/fundraising-frontend-content": "dev-master as 1.0.x-dev"
```

Comment:
Mainly because (for the moment = dev/prototyping) we are tracking the master instead of a tag, which is not the recommended way of using package management.